Update runner-installation.adoc (#5105)
circleci/circleci-docs: jekyll/_cci2/runner-installation.adoc

NOTE: The default value is 5 hours.

===== Customizing job timeouts and drain timeouts

> If you would like to customize the job timeout setting, you can "drain" the job by sending the Launch Agent a termination (TERM) signal, which then causes the Launch Agent to attempt to gracefully shutdown. When this TERM signal is received, the launch agent enters 'draining' mode, preventing the Launch Agent from accepting any new jobs, but still allowings any current active job to be completed. At the end of "draining," the Launch Agent then signals the Task Agent to cancel any active job (by sending it a TERM signal).

Reviewer: ```suggestion
If you would like to customize the job timeout setting, you can "drain" the job by sending the Launch Agent a termination (TERM) signal, which then causes the Launch Agent to attempt to gracefully shutdown. When this TERM signal is received, the launch agent enters 'draining' mode, preventing the Launch Agent from accepting any new jobs, but still allowing any current active job to be completed. At the end of "draining," the Launch Agent then signals the Task Agent to cancel any active job (by sending it a TERM signal).
```

> NOTE: If the Task Agent does not exit after a brief period, the Launch Agent will manually kill it by sending it a KILL signal.

Reviewer: ```suggestion
NOTE: If the Task Agent does not exit a brief period after the TERM, the Launch Agent will manually kill it by sending it a KILL signal.
```

Draining can end in one of two ways:

* The task has been in the draining state for longer than the configured `max_run_time`.

> * A TERM signal is received by the Launch Agent during "draining".

Reviewer: ```suggestion
* An additional TERM signal is received by the Launch Agent during "draining".
```
